### PR TITLE
docs/install : proxy installation

### PR DESCRIPTION
**Pull request by srisankethu**
Fixes https://github.com/coala-analyzer/coala/issues/1715

### CODE REVIEW

**Comment by sils1297** [comment]
unack 413231 - almost there

**Comment by sils1297** [comment]
rebase neede dbecause I merged something else

File: docs/Users/Install.rst [hunk 1]
```diff
@@ -153,6 +153,18 @@ If you want to install coala to an alternate location you can e.g. call
...
 
+.. note::
+
+    If you are using a proxy, follow these steps:
+
+    -  Set up your system-wide proxy.
+    -  Use ``sudo -E pip install coala``(the ``-E`` flag takes the
+       existing environment variables into the ``sudo`` environment).
+
+    Read
```
Comment (sudheesh001) [inline docs/Users/Install.rst]: Please rephrase this to `You could also set your pip.conf file to use a proxy, to know more read ....`

**Comment by srisankethu** [comment]
@sils1297 Made changes!

**Comment by sils1297** [comment]
ack ea65083

**Comment by sils1297** [comment]
@rultor merge
